pkgs/development/libraries/boxfort/default.nix

```diff
@@ -1,31 +1,35 @@
-{ stdenv, fetchFromGitHub, cmake, pkg-config, gettext, libcsptr, dyncall
-, nanomsg, python37Packages }:
+{ stdenv
+, fetchFromGitHub
+, git
+, meson
+, ninja
+, python37Packages
+}:
 
 stdenv.mkDerivation rec {
-  version = "unstable-2019-09-19";
+  version = "unstable-2019-10-09";
   pname = "boxfort";
 
   src = fetchFromGitHub {
     owner = "Snaipe";
     repo = "BoxFort";
-    rev = "926bd4ce968592dbbba97ec1bb9aeca3edf29b0d";
-    sha256 = "0mzy4f8qij6ckn5578y3l4rni2470pdkjy5xww7ak99l1kh3p3v6";
+    rev = "356f047db08b7344ea7980576b705e65b9fc8772";
+    sha256 = "1p0llz7n0p5gzpvqszmra9p88vnr0j88sp5ixhgbfz89bswg62ss";
   };
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ cmake pkg-config ];
-
-  buildInputs = [
-    dyncall
-    gettext
-    libcsptr
-    nanomsg
+  nativeBuildInputs = [
+    git
```

Comment:
Is git necessary for something, seems to build fine without it. `src` does not contain the `.git` directory anyway.